virtual_rainforest/core/core_schema.json

```diff
@@ -9,23 +9,44 @@
                "description": "Details of the grid to configure",
                "type": "object",
                "properties": {
-                  "nx": {
+                  "grid_type": {
+                     "description": "The grid cell type",
+                     "type": "string",
+                     "default": "square"
+                  },
+                  "cell_area": {
+                     "description": "The area of each grid cell",

```

Comment:
Is it worth mentioning a unit here?

Reply:
Yup - also added some clarification in `core.grid` docstrings